app-ktor-server/src/jvmMain/kotlin/com/ustadmobile/lib/rest/ContentEntryLinkImporter.kt

```diff
@@ -0,0 +1,60 @@
+package com.ustadmobile.lib.rest
+
+import com.ustadmobile.core.db.UmAppDatabase
+import com.ustadmobile.door.ext.DoorTag
+import com.ustadmobile.lib.contentscrapers.abztract.ScraperManager
+import com.ustadmobile.lib.db.entities.ContentEntryWithLanguage
+import io.ktor.application.call
+import io.ktor.http.HttpStatusCode
+import io.ktor.request.receive
+import io.ktor.response.respond
+import io.ktor.routing.Route
+import io.ktor.routing.post
+import io.ktor.routing.route
+import org.apache.http.HttpStatus
+import org.kodein.di.instance
+import org.kodein.di.ktor.di
+import org.kodein.di.on
+
+@ExperimentalStdlibApi
+fun Route.ContentEntryLinkImporter() {
+
+    route("import") {
+
+        post("validateLink") {
+
+            val url = call.receive<String>()
```

Comment:
Posting a string as the body seems a bit strange. Post body is normally used for larger payloads. Why not use a normal query parameter for this? (e.g. validateLink?url=...)